migrations/sql/01_init.up.sql

```diff
@@ -18,8 +19,21 @@ CREATE TABLE IF NOT EXISTS metrics
     -- Custom labels
     `labels.key` Array(String) COMMENT 'Custom labels names',
     `labels.value` Array(String) COMMENT 'Custom labels values',
-    agent_uuid String COMMENT 'Identifier of agent that collect and send metrics',
-    metrics_source Enum8('METRICS_SOURCE_INVALID' = 0, 'MYSQL_SLOWLOG' = 1, 'MYSQL_PERFSCHEMA' = 2, 'MONGODB_PROFILER' = 3) COMMENT 'Source of metrics: slowlog, perf schema, etc.',
+    agent_id String COMMENT 'Identifier of agent that collect and send metrics',
+    agent_type Enum8(
+        'agent_type_invalid' = 0,
+        'pmm-agent' = 1,
+        'node_exporter' = 2,
+        'mysqld_exporter' = 3,
+        'mongodb_exporter' = 4,
+        'postgres_exporter' = 5,
+        'qan-mysql-perfschema-agent' = 6,
+        'qan-mysql-slowlog-agent' = 7,
+        'qan-mongodb-profiler-agent' = 8,
+        'rds_exporter' = 9,
+        'proxysql_exporter' = 10,
+        'external_exporter' = 11
```

Comment:
I think you can keep only QAN types there.

Reply:
yeah, updated.